client/src/App.js

```diff
@@ -43,6 +44,7 @@ const App = () => {
         <BrowserRouter>
           <ProgressBar />
           <Navbar />
+          <Footer />
           <Switch>
             <Route path="/" exact component={Landing} />
             <Route path="/login" component={LoginPage} />
```

Comment:
You should put this at the bottom of the tree, right above </BrowserRouter>

Reply:
moved it